src/containers/ServiceFacilityForm.js

```diff
@@ -0,0 +1,127 @@
+/** @jsx jsx */
+import { useState, useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { jsx } from '@emotion/core';
+import AutosuggestHighlightMatch from 'autosuggest-highlight/match';
+import AutosuggestHighlightParse from 'autosuggest-highlight/parse';
+import { useHistory } from 'react-router-dom';
+
+import { Routes } from 'constants/Routes';
+import { routeWithParams } from 'lib/utils/routes';
+
+import Text from 'components/Text';
+import { TEXT_TYPE } from 'components/Text/constants';
+import Form from 'components/Form';
+import Autosuggest from 'components/Autosuggest';
+
+import * as tools from '../functions';
+// service todo: update this list with service organizations instead of list of hospitals
```

Comment:
if i'm not mistaken, i think the facility form should still use hospitals for its data - this is just to verify which hospital a healthcare worker is coming from